### DISCUSSION TITLE
Jaeger Datasource - Allow 'serviceName' and 'operationName' to be included in Trace to Metric Tags

### DISCUSSION THREAD

**Discussion by hannahchan** [grafana/grafana#54912] [Data Sources]
I've been playing around with the Trace To Metrics feature in Grafana which is currently behind a feature toggle. I tried to get this working my Jaeger datasource however I found this unworkable for my scenario.

I'm currently using an OpenTelemetry Collector to generate span metrics from my traces. The span metrics have only four dimensions and they are;

- operation
- service_name
- span_kind (internal, client, service)
- status_code (UNSET, OK, ERROR)

From Grafana in my Jaeger Datasource, I'm unable to map `operation` and `service_name` to a value in the span attributes because Jaeger doesn't include them there. The trace to metrics feature currently only allow span attributes to be mapped to tags.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
